operator/apis/loki/v1/lokistack_types.go

```diff
@@ -566,9 +565,12 @@ type ObjectStorageSchema struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:select:v11","urn:alm:descriptor:com.tectonic.ui:select:v12","urn:alm:descriptor:com.tectonic.ui:select:v13"},displayName="Version"
 	Version ObjectStorageSchemaVersion `json:"version"`
 
-	// EffectiveDate is the date in UTC that the schema will be applied on.
-	// To ensure readibility of logs, this date should be before the current
-	// date in UTC.
+	// EffectiveDate contains a date in YYYY-MM-DD format which is interpreted in the UTC time zone.
+	//
+	// The configuration always needs at least one schema that is currently valid. This means that when creating a new
+	// LokiStack it is recommended to add a schema with the latest available version and an effective date of "yesterday".
```

Comment:
@xperimental perhaps instead of 

`LokiStack it is recommended to add a schema with the latest available version and an effective date of "yesterday".`  by

`LokiStack it is recommended to add a schema with the latest available version and it should have an effective date of "yesterday".`? 

For being clear that the "recommended" is only for the schema version, but for the effective date, it should/must from yesterday for new installations